ci/assign-labels.sh

```diff
@@ -33,8 +33,8 @@ label_issue() {
     # Remove existing mutually-exclusive labels, if any.
     if is_label_exclusive "$new_label"; then
       local i
```

Comment:
is this needed now?

Reply:
It does not appear to be referenced within the context.  Shellcheck did not pick this up.

Reply:
Removed in new commit.